.github/workflows/pull-request.yml

```diff
@@ -240,19 +240,76 @@ jobs:
 
       - name: resolve dependency
         run: npm install -g @graphql-inspector/cli
-      
+
       - name: Clone API Repository
         run: |
           # Retrieve the complete branch name directly from the GitHub context
           FULL_BRANCH_NAME=${{ github.base_ref }}
           echo "FULL_Branch_NAME: $FULL_BRANCH_NAME"
-          
+
           # Clone the specified repository using the extracted branch name
-          git clone --branch $FULL_BRANCH_NAME https://github.com/PalisadoesFoundation/talawa-api && ls -a 
+          git clone --branch $FULL_BRANCH_NAME https://github.com/PalisadoesFoundation/talawa-api && ls -a
 
       - name: Validate Documents
         run: graphql-inspector validate './src/GraphQl/**/*.ts' './talawa-api/schema.graphql'
 
+  Docker-Start-Check:
```

Comment:
Please ensure that this does not run if the PR is submitted by the dependabot user

Reply:
Yes, I did it. 
![image](https://github.com/user-attachments/assets/b90a2e1a-01ce-4a59-ab17-74483b6d6104)